### DISCUSSION TITLE
Turn Off Title and Text on Launched Apps

### DISCUSSION THREAD

**Discussion by mogulman52** [valinet/ExplorerPatcher#737] [General]
I successfully added Quick Launch back in Taskbar.   Disabled Show Title and Show Text on Quick Launch.  Looks perfect.  However, when I launch an app the Title and Text is showing.  VERY ANNOYING.  I see no way to turn it off.

**Comment by valinet** [comment]
https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems

https://github.com/valinet/ExplorerPatcher/wiki/All-features#combine-taskbar-icons-on-primarysecondary-taskbars

**Comment by jdorlon-Quest** [comment]
![image](https://user-images.githubusercontent.com/54901755/151992935-9ee1e946-1e23-41bc-8bc2-f87a53fa4dd7.png)
